healthcheck/healthchecksTest.yml

```diff
@@ -0,0 +1,17 @@
+name: rhobot healthcheck test
+distribution:
+- stefan.fox@cfpb.gov
```

Comment:
I wonder... would it be better to externalize things like email addresses? I'm not sure you want that kind of thing in code, especially public. What do you think?